cl/api/templates/case-law-api-docs-vlatest.html

```diff
@@ -148,6 +148,9 @@ <h3 id="opinion-endpoint">Opinions <small> — <code>{% url "opinion-list" versi
   </p>
   <p>As with all other APIs, you can look up the field descriptions, filtering, ordering, and rendering options by making an <code>OPTIONS</code> request:
   </p>
+  <p class="alert alert-warning"><i class="fa fa-warning"></i> <strong>Listen Up:</strong> When retrieving opinion text, prefer the <code>html_with_citations</code> field instead of <code>plain_text</code>. The <code>html_with_citations</code> contains the raw text of the decision is the most reliable field for search.<br><br>
```

Comment:
```suggestion
  <p class="alert alert-warning"><i class="fa fa-warning"></i> <strong>Listen Up:</strong> When retrieving opinion text, prefer the <code>html_with_citations</code> field instead of <code>plain_text</code>. The <code>html_with_citations</code> field contains the raw text of the decision, and is the most reliable field for most purposes.<br><br>
```